solidity/migrations/2_deploy_contracts.js

```diff
@@ -128,4 +129,13 @@ module.exports = async function(deployer, network) {
     KeepRandomBeaconOperatorStatistics,
     KeepRandomBeaconOperator.address
   );
+
+  if(network === 'local') {
+    await deployer.deploy(
+        BeaconRewards,
+        KeepToken.address,
+        KeepRandomBeaconOperator.address,
+        TokenStaking.address,
+    )
```

Comment:
Shouldn't this be deployed for development and Ropsten networks as well?

Reply:
If we could, it would be great! Since the app would probably crash on ropsten version(because we don't deploy rewards contract to ropsten network) and we would have to implement a workaround only for ropsten version of the dashboard.

Reply:
Yeah, I think cc @pdyraga and let's not block the merge on it, but it makes sense to me to deploy these contracts on Ropsten, even if we leave them dormant somehow.

Reply:
Agreed, it should be deployed to Ropsten. We were not doing an RC cut for reward contracts and that's the only reason why rewards contracts are not yet on Ropsten. I vote for removing the condition `if(network === 'local')` so that it all goes to Ropsten during the next RC cut.